app/main.py

```diff
@@ -2,0 +1,51 @@
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
```

Comment:
Appending to `self.skills` will modify the class variable `skills` shared across all instances. If `skills` is intended to be an instance-specific attribute, it should be initialized in the `__init__` method of `SoftwareEngineer`.